### ISSUE TITLE
Individual country/sector/agency page updates

### ISSUE THREAD

**Issue by anjesh** [younginnovations/UN-Transparency-Portal#198]
## Why

* to improve the page info structure

## What

- [ ] Remove ‘Currently publishing in IATI’ text and space to move the boxes up
- [ ] add "filter by year" option. Need to think where this would go.
- [ ] default should be current calendar year, so 2018 for now

## Notes

The lowest drill down on the site is the country page, or individual agency or sector page. I think this page is rather well constructed and offers plenty of opportunity for further analysis by the user. I would ask the ‘Currently publishing in IATI’ line and space be removed as to move the boxes up. The boxes contain good data but I want the ability to filter by the year. Have it defaulted to 2018 and update as the user selects a different year. The exact same way as the suggested change for the homepage.

I’m fine with the tables and map below and the ability for a user to drilldown to the project level. You can leave that and the blue box at the bottom.

![image](https://user-images.githubusercontent.com/120488/36012457-505ab5ae-0d86-11e8-9dc1-e0da45f76947.png)

**Comment by bezunakarmi** [comment]
@forsythej  changes have been implemented.

**Comment by forsythej** [comment]
Can we add one thing to the individual country page. After speaking with Gina, it would be best to have a table that displays current year expenditure and future budgets for Sectors. The table should show only sectors that actually have values, not an entire list of all Sectors in the codelist. 

Let me know if this is feasible and, if so, a time frame when we can expect it.

@bezunakarmi @anjesh

**Comment by anjesh** [comment]
@forsythej We might have missed your last comment. Do you mean we list http://open.undg.org/ctrack.html?country=AF&tongue=eng#view=sectors in the country page (but with columns for 2018 expenditure, and budgets for 2019, 2020, upto how many years)

**Comment by forsythej** [comment]
Yes, exactly.  Could you make another table, similar to the existing Sector table on the country page, that only displays sectors with forward looking budgets?

**Comment by anjhero** [comment]
Here's the design for the sector table in the country detail page:
![undg-open-ui](https://user-images.githubusercontent.com/528549/38611944-e86a6fda-3da4-11e8-8e3e-14132468a21e.png)
Since the page is already long, we've fixed the maximum height of the table. If/when the table is longer, the user can scroll within the table block.

**Comment by anjesh** [comment]
@forsythej please confirm you want something like this in the country page before we start making changes in the code.

**Comment by forsythej** [comment]
@anjesh Excellent. That's the data I want visualized. Is it possible to put it in a bar graph just like the 'Where does the money go? (2018)' with the heading 'Where the money will go (dates)'?

**Comment by anjesh** [comment]
This gets tricky and has lots of moving parts. 
@forsythej see if you still like to have that sector detail table in country page http://undg.staging.yipl.com.np/ctrack.html?country=AF&tongue=eng&detailYear=all&year=2019#view=main 
Please ignore some design bugs, see if that sector detail table is ok to move forward. I would like to drop this feature, looking at lots of details in this page.

cc @bezunakarmi
